FontAtlasProsessor.go

```diff
@@ -40,7 +40,15 @@ func (f *FontInfo) String() string {
 func (f *FontInfo) SetSize(size float32) *FontInfo {
 	result := *f
 	result.size = size
+
+	for _, i := range extraFonts {
+		if i.String() == result.String() {
+			return &result
+		}
+	}
+
```

Comment:
I'm not happy about it but currently no idea how to baypass it...
the pint is that I need to check if the font is already in extra fonts before asking font atlas to be built